autoload/utils/exec/job.vim

```diff
@@ -44,6 +44,15 @@ function! s:vimClose(channel) abort
     call s:createQuickFix()
     call s:closeBuffer()
 
+    if has_key(s:cmake4vim_job, 'job')
+        if job_info(s:cmake4vim_job['job'])['exitval'] == 0
+            call utils#common#Warning('Success!')
+        else
+            call utils#common#Warning('Failure!')
```

Comment:
I think `echon` should work better in this case, because `utils#common#Warning()` uses the red color and highlighted that something happened.